lib/common/db-renderable-content.js

```diff
@@ -212,7 +212,9 @@ function dbRenderableContentServiceFactory(
                 })
                 .spread(function(hash, data) {
                     if (template.hash !== hash) {
-                        throw new Errors.InternalServerError('template meta error');
+                        throw new Errors.InternalServerError(
+                            'hash mismatch for ' + name + ' in ' + scope + ' scope'
```

Comment:
maybe not, but I'll ask:

`template hash mismatch ...` instead of `hash mismatch ...`?

Totally good either way